programs/drift/src/state/amm_cache.rs

```diff
@@ -53,11 +51,12 @@ pub struct CacheInfo {
     pub amm_inventory_limit: i64,
     pub oracle_price: i64,
     pub oracle_slot: u64,
+    pub market_index: u16,
```

Comment:
are these already going to be set for some of the entries? if so, needs to be bottom of struct above padding